circonus/circonus_test.go

```diff
@@ -99,6 +99,34 @@ func TestSetGauge(t *testing.T) {
 	}
 }
 
+func TestSetPrecisionGauge(t *testing.T) {
+	q := make(chan string)
+
+	server := fakeBroker(q)
+	defer server.Close()
+
+	cfg := &Config{}
+	cfg.CheckManager.Check.SubmissionURL = server.URL
+
+	cs, err := NewCirconusSink(cfg)
+	if err != nil {
+		t.Errorf("Expected no error, got '%v'", err)
+	}
+
+	go func() {
+		cs.SetPrecisionGauge([]string{"foo", "bar"}, 1)
```

Comment:
Is it worth testing this with a value that can't be represented in a float32 to be sure it actually works as expected?